backend/fms_core/viewsets/imported_file.py

```diff
@@ -45,7 +50,7 @@ def download(self, _request, pk) -> Response:
         try:
             with open(file_path, "rb") as file:
                 response = HttpResponse(content=file)
-                response["Content-Type"] = mime_type_guess[0]
+                response["Content-Type"] = mime_type_guess[0] if mime_type_guess[0] != None else 'application/octet-stream'
```

Comment:
We could also replace the mime_type_guess and send everything as an octet-stream and let the frontend sort.